app/pubspec.lock

```diff
@@ -514,6 +668,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "4.5.1"
+  yaml:
+    dependency: transitive
+    description:
+      name: yaml
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "2.2.1"
 sdks:
-  dart: ">=2.10.0-110 <2.11.0"
-  flutter: ">=1.16.0 <2.0.0"
+  dart: ">=2.10.0 <2.11.0"
+  flutter: ">=1.22.0 <2.0.0"
```

Comment:
The minimum should definitely be 1.16, unless your newest plugin changed this. I think not though, because you have the camera dependencies above which we aren't using anymore.

Reply:
Once I deleted the camera in our dependencies, removed the publock and finally run flutter pub get, the min changed to 1.16